sklearn/mixture/gmm.py

```diff
@@ -662,6 +662,15 @@ def aic(self, X):
 @deprecated("The class GMM is deprecated in 0.18 and will be "
             " removed in 0.20. Use class GaussianMixture instead.")
 class GMM(_GMMBase):
+    """
+    Legacy Gaussian Mixture Model
+
+    .. deprecated:: 0.18
+        The :class:`sklearn.mixture.GMM` will be removed in 0.20.
```

Comment:
Same comment here. Could we simply have it as `This class will be removed in 0.20` please? :)